mesos_master/tests/conftest.py

```diff
@@ -0,0 +1,23 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+import copy
+import pytest
+
+from datadog_checks.dev import docker_run
+
+from . import common
+
+
+@pytest.fixture(scope='session')
+def dd_environment(instance):
+    compose_file = os.path.join(common.HERE, 'compose', 'docker-compose.yml')
+
+    with docker_run(compose_file):
```

Comment:
Is there a log line or endpoint we could check from mesos-master to confirm everything came up properly?